odoo/addons/base/models/res_currency.py

```diff
@@ -275,7 +275,24 @@ class CurrencyRate(models.Model):
         ('currency_rate_check', 'CHECK (rate>0)', 'The currency rate must be strictly positive.'),
     ]
 
+<<<<<<< Updated upstream
```

Comment:
Wrong `git stash pop` but you can make it work...

Another way to do it would be to do the conversion rates each time you set/retrieve the rate thanks to two computed fields with inverse in the `res.currency.rate`, which would be less understandable IMO